src/store/CategoricalDisplayStore.ts

```diff
@@ -43,6 +43,18 @@ export default class CategoricalDisplayStore {
     searchInputClassName?: string;
     placeholderText: string;
 
+    isShown = false;
+    scrollPanel?: HTMLElement | null;
```

Comment:
```suggestion
    scrollPanel?: HTMLElement;
```
nullable type ? should be enough to cater for null

Reply:
for this and other TS modifications, I will add a separate card because one of our tasks for next quarter is to change Stores structure and we will cover these changes with them as well.